### ISSUE TITLE
bug in findClosingIndex

### ISSUE THREAD

**Issue by ttghr** [alabianca/xml-to-json#6]
with
`    const openinTag   = tag.replace('</', '<').replace('>', '');`
you will not find the next opening tag in this xml

```
<aaa>
	<bbb>
		<ccc>ccc</ccc>
		<bbb1>bbb1</bbb1>
	</bbb>
	<bbb>
		<ddd>ddd</ddd>
		<bbb2>bbb2</bbb2>
	</bbb>
</aaa>
```

Parsing output: Invalid XML Error

the following fix works for my case:
`    const openinTag   = tag.replace('</', '<');`

**Comment by alabianca** [comment]
Looking into it. I just replaced the bb1 and bb2 tags with yyy and it parsed fine. Not sure yet why. Will run your fix through the tests

**Comment by alabianca** [comment]
@ttghr I have a fix on branch `xmlToJsonStream/patch/1.0.5` . The issue was indeed in `findClosingIndex`, but was related to not properly distinguishing between `<bbb><bbb1></bbb1></bbb>` It treated `<bbb1>` as a `<bbb>` tag if it was nested. I included your scenario in the jasmine tests and added some additional similar test cases and it all passed so far. I will publish this fix soon

**Comment by alabianca** [comment]
fixed in v1.0.5
